### PR TITLE
Notifications docs

### PR DESCRIPTION
**Pull request by gary149**
Add this to document a bit on the new notification system.
edit: adding spec as it can help for the review: https://www.figma.com/file/6ffc9hUNQzH8M8wiEQZH7p/Notifications?node-id=43%3A2

### CODE REVIEW

**Review by coyotte508 — APPROVED** [review]

File: docs/hub/notifications.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# Notifications
+
+Notifications allow you to know when new activities (pull requests or discussions) happen on users or organizations you are watching.
```
Comment (coyotte508) [inline docs/hub/notifications.md:3]:
```suggestion
Notifications allow you to know when new activities (pull requests or discussions) happen on models, datasets and spaces belonging to users or organizations you are watching.
```

File: docs/hub/notifications.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+By default, you'll be watching all the organizations you are a member of and will be notified of any new activity on those.
+
+You can also choose to get notified on arbitrary users or organizations, to do so use the "Watch" button on their HF profiles. Note that you can also quickly watch/unwatch users and organizations directly from the [notifications settings](#notifications-settings).
+
+_Unlike Github, you cannot watch a specific repository. You must watch users/organizations to get notified about any new activity on any of their repositories. The goal is to simplify this functionality for users as much as possible and to make sure you don't miss anything you might be interested in._
+
+## Notifications settings
+
+In your [notifications settings](https://huggingface.co/settings/notifications) page, you can choose specific channels to get notified on depending on the type of activity, for example, receiving an email for direct mentions but only a web notification for new activity on watched users and organizations. By default, you'll get an email and a web notification for any new activity but feel free to adjust your settings depending on your needs.
```
Comment (coyotte508) [inline docs/hub/notifications.md:25]:
```suggestion
In your [notifications settings](https://huggingface.co/settings/notifications) page, you can choose specific channels to get notified on depending on the type of activity, for example, receiving an email for direct mentions but only a web notification for new activity on watched users and organizations. By default, you'll get an email and a web notification for any new activity but feel free to adjust your settings depending on your needs.

Clicking the unsubscribe link in an email will unsubscribe you for the type of activity, eg direct mentions.
```

File: docs/hub/notifications.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
+# Notifications
+
+Notifications allow you to know when new activities (pull requests or discussions) happen on models, datasets and spaces belonging to users or organizations you are watching.
```
Comment (osanseviero) [inline docs/hub/notifications.md:3]:
```suggestion
Notifications allow you to know when new activities (Pull Requests or discussions) happen on repositories belonging to users or organizations you are watching.
```

File: docs/hub/notifications.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+
+By default, you'll receive a notification if:
+
+- Someone mentions you in a discussion/PR.
+- A new comment is created in a discussion/PR you participated in.
+- A new discussion/PR, or comment is created in one of the repositories of an organization or user you are watching.
```
Comment (osanseviero) [inline docs/hub/notifications.md:9]:
```suggestion
- A new discussion/PR or comment is created in one of the repositories of an organization or user you are watching.
```

File: docs/hub/notifications.md [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+
+![Notifications page](https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/hub/notifications-page.png)
+
+You'll get new notifications by email and [directly on the website](https://huggingface.co/notifications), you can change this in your notifications settings.
```
Comment (osanseviero) [inline docs/hub/notifications.md:13]: Maybe add a link to the notification settings at the end

File: docs/hub/notifications.md [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+
+## Watching users and organizations
+
+By default, you'll be watching all the organizations you are a member of and will be notified of any new activity on those.
+
+You can also choose to get notified on arbitrary users or organizations, to do so use the "Watch" button on their HF profiles. Note that you can also quickly watch/unwatch users and organizations directly from the [notifications settings](#notifications-settings).
```
Comment (osanseviero) [inline docs/hub/notifications.md:19]:
```suggestion
You can also choose to get notified on arbitrary users or organizations. To do so, use the "Watch" button on their HF profiles. Note that you can also quickly watch/unwatch users and organizations directly from the [notifications settings](#notifications-settings).
```

**Review by osanseviero — APPROVED** [review]
Thanks for this!! :fire:

File: docs/hub/notifications.md (hunk 3 above)
Comment (julien-c) [reply docs/hub/notifications.md:3]: (nit) i think it's fine to keep "models, datasets and Spaces" to be more ML specific

File: docs/hub/notifications.md [hunk 7]
```diff
@@ -0,0 +1,32 @@
+# Notifications
+
+Notifications allow you to know when new activities (pull requests or discussions) happen on models, datasets and spaces belonging to users or organizations you are watching.
+
+By default, you'll receive a notification if:
+
+- Someone mentions you in a discussion/PR.
+- A new comment is created in a discussion/PR you participated in.
```
Comment (julien-c) [inline docs/hub/notifications.md:8]:
```suggestion
- A new comment is posted in a discussion/PR you participated in.
```

File: docs/hub/notifications.md [hunk 8]
```diff
@@ -0,0 +1,32 @@
...
+- A new discussion/PR, or comment is created in one of the repositories of an organization or user you are watching.
+
+![Notifications page](https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/hub/notifications-page.png)
+
+You'll get new notifications by email and [directly on the website](https://huggingface.co/notifications), you can change this in your notifications settings.
+
+## Watching users and organizations
+
+By default, you'll be watching all the organizations you are a member of and will be notified of any new activity on those.
```
Comment (julien-c) [inline docs/hub/notifications.md:17]:
```suggestion
By default, you'll be watching all the organizations you are a member of and will be notified of any new activity on those.

**This is different from services like GitHub where you watch individual repos, here you watch users or organizations (and as a consequence you're notified on activity on their respective repos).**
```

Comment (julien-c) [reply docs/hub/notifications.md:17]: Oh ok no read too fast, you mention this below already

**Review by julien-c — APPROVED** [review]
really cool, thanks

**Comment by julien-c** [comment]
Maybe we can update or add the following screenshot to make it clear that the two sections are directly related

notifications-settings

**Pull request merged**